### ISSUE TITLE
[BUG] built-in RTE reset styles on rte:disable

### ISSUE THREAD

**Issue by h3llomoto** [artf/grapesjs#2707]
Hello, @artf , 
I've catch a bug at built-in RTE:
- added standart email text element with a red styled tag a inside:
```
<div>
  <span>
    Lorem ipsum dolor sit amet 
    <a href="https://google.com" target="_blank" style="color: #f00">red link</a>
  </span>
</div>
```
- enable rte by doubleclick on span element
- disable rte by deselecting element
- the link is blue now

It adds "-2" to link's id after every time I deselect it.
I think the problem starts at dom_components/view/ComponentView.js:318, updateContent function.
GrapesJS 0.16.4

**Comment by h3llomoto** [comment]
Hey, is there anybody?

**Comment by artf** [comment]
@h3llomoto PRs are welcome buddy

**Comment by artf** [comment]
Fixed and ready for the next release #3069
